core/modules/meet/client/meet.js

```diff
@@ -97,7 +97,8 @@ meetHighLevel = {
     if (meet.api) return;
 
     const user = Meteor.user();
-    const currentZone = zoneManager.currentZone();
```

Comment:
it is `const currentZone = zoneManager.currentZone();` that you have to keep here, `zones` doesn't exist anymore

Reply:
Updated 🙌

Reply:
Thank you!